tests/expected/elm/Client.elm

```diff
@@ -411,6 +412,35 @@ updateMeRequest prefix maybeTimeout withCredentials updateUser =
         }
 
 
+{-| Contains a "get" request to the endpoint: /history, to be sent with Http.send
+    
+    The User Activity endpoint returns data about a user's lifetime activity with Uber. The response will
+    include pickup locations and times, dropoff locations and times, the distance of past requests, and
+    information about which products were requested.
+-}
+historyRequest : String -> Maybe Time.Time -> Bool -> Maybe Int -> Maybe Int -> Http.Request Activities
```

Comment:
The `Activities` seem to be a return type -- is this directly related to the feature change?

Could you maybe add a test case (a separate swagger def) that tests for non-JSON return type?

Reply:
I will add a test case for that!